platform/core/flatbuffers_shared/generate_flatbuffers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e    # Exit if any command fails
+
+echo '===> Compiling the Schema...'
+
+cd flatbuffers_shared
```

Comment:
Right, so you're running it from the upper directory... and then changing to the directory where it resides... Again, putting this in a Makefile will avoid this kind of things... Alternatively, you can just put the Makefile in this directory and instruct the user to run it there.